docs/agent-api.asciidoc

```diff
@@ -787,3 +787,24 @@ This enables log correlation to APM traces with text-only loggers.
 "trace.id=abc123 span.id=abc123"
 ----
 // end::currentTraceIds[]
+
+[[apm-register-custom-metrics]]
+==== `apm.registerCustomMetric(name, callback, labels)`
+
+[small]#experimental#
```

Comment:
```suggestion
experimental::[]
```

This is special markup that our docs build will expand into a standard disclaimer: https://github.com/elastic/docs#experimental-and-beta

Reply:
Didn't notice that. Updated.